src/app/GitUI/UserControls/TreeViewExtensions.cs

```diff
@@ -71,6 +84,15 @@ public static void RestoreExpandedNodesState(this TreeNode node, HashSet<string>
         }
     }
 
+    public static void ScrollLeftMost(this TreeView? treeView)
```

Comment:
Is there a reason to keep this public?

Reply:
> Is there a reason to keep this public?

Yes, I am also calling this after setting `TreeView.TopNode` (in #12116).